tests/testcafe/tests.js

```diff
@@ -50,6 +50,35 @@ function prepareClient() {
   document.querySelector('base').parentNode.removeChild(document.querySelector('base'));
 }
 
+function hiddenTest(game, variant, md5, tests) {
+  const testUrl = `${server}/library/${game}.vtt#VTT`;
+  test.after(async t => {
+    await removeGame(t);
+    await t.expect(Selector('#statesOverlay').visible).ok();
+  })(`Public library: ${game} (variant ${variant})`, async t => {
+    await ClientFunction(prepareClient)();
+    await t
+      .pressKey('esc')
+      .click('#statesButton')
+      .setNativeDialogHandler(() => testUrl, { dependencies: { testUrl }})
+      .click('button.prettyButton.link:nth-child(2)')
+      .pressKey('enter')
```

Comment:
The first line already adds the game. The second line adds it again which is the reason why one of those two games was still there when running TestCafé again.

You can see this easily by running `npm run testcafe-firefox -- --test-grep '.*Test_Room.*' --debug-mode`.

Reply:
I thought the .click would just open up the 'Enter Link" dialog box, and that enter would be required after the text was entered.  But if it is working, then I was wrong.